HATE/Shuffle.cs

```diff
@@ -271,37 +270,37 @@ public static bool SimpleShuffler(UndertaleChunk _chunk, UndertaleData data, Ran
         return true;
     }
 
-    public static bool ShuffleAudio2_Shuffler(UndertaleChunk _chunk, UndertaleData data, Random random, float shufflechance, StreamWriter logstream, bool _friskMode)
+    public static bool ShuffleAudio2_Shuffler(UndertaleChunk _chunk, UndertaleData data, Random random, float shuffleChance, StreamWriter logStream, bool friskMode)
     {
         IList<UndertaleString> _pointerlist = (_chunk as UndertaleChunkSTRG)?.List;
-        IList<UndertaleString> strgClone = new List<UndertaleString>(_pointerlist);
+        IList<UndertaleString> stringClone = new List<UndertaleString>(_pointerlist);
```

Comment:
The variable naming was referring to the chunk but fine

Reply:
Ah, thought it was supposed to be short for string.